src/Serializer/Filter/GroupFilter.php

```diff
@@ -57,7 +58,7 @@ public function apply(Request $request, bool $normalization, array $attributes,
     public function getDescription(string $resourceClass): array
     {
         return [
-            $this->parameterName.'[]' => [
+            "{$this->parameterName}[]" => [
```

Comment:
This looks less clear to me

Reply:
Dunno why I changed this actually. Reverted.